docs/dietpi_tools.md

```diff
@@ -762,6 +774,16 @@ dietpi-backup
 
     The system stores these settings in the files `/boot/dietpi/.dietpi-backup_settings` and `/boot/dietpi/.dietpi-backup_inc_exc`, which are generated from `dietpi-backup` automatically. Therefore, the files do not need to be changed manually by the user.
 
+=== "Scripted run"
+
+    DietPi-Backup can be run from the command line or from scripts without user interaction by calling it via 
+    
+    ```sh
+    dietpi-backup 1
+    ```
+
+    A similar restore procedure is not recommended to avoid accidentally system overwrites.
```

Comment:
Indeed not recommended to do this in non-interactive script, cron jobs or such, but shall we add the info for completeness, e.g. just to skip the clicks through the menu and call the restore directly from console?
```suggestion
    A similar restore procedure is not recommended to avoid accidentally system overwrites. But to skip navigating through the menu, it can be done as well via

    ```sh
    dietpi-backup -1
    ```
```